src/ConfigUpdater.php

```diff
@@ -0,0 +1,279 @@
+<?php
+
+namespace Drupal\commerce;
+
+
+use Drupal\Component\Utility\Crypt;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Config\ExtensionInstallStorage;
+use Drupal\Core\Config\InstallStorage;
+use Drupal\Core\Config\StorageInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+
+/**
+ * Class ConfigUpdater.
```

Comment:
We usually document services like this with "Default implementation of the ConfigUpdaterInterface.", then place the real docs on the ConfigUpdaterInterface.